packages/renderer/src/lib/dashboard/ProviderConfiguring.svelte

```diff
@@ -86,7 +86,7 @@ onDestroy(() => {
 });
 </script>
 
-<div class="p-2 flex flex-col bg-charcoal-800 rounded-lg">
+<div class="p-2 flex flex-col bg-charcoal-800 rounded-lg" role="region" aria-label="{provider.name} Status">
```

Comment:
How about we use `aria-label=${provider.name} Provider` notation, since these objects (big tiles) can also do some actions (install, check, run and show version) and not only provide a status.

Reply:
Please consider that change in every `Provider*` file